scripts/utils.sh

```diff
@@ -93,28 +93,29 @@ function get_main_ip_v6() {
 }
 
 function wait_for_url_and_run() {
+    URL="$1"
+    FUNCTION="$2"
     RETRIES=15
```

Comment:
Thanks, then I think it will mostly depend on how fast the container image will be downloaded from the registry.

In the `e2e-metal-assisted` jobs , the script retried 10 times.
 
I would set it up to 1min just to be on the safe side and because the machine may be located in another region. What do you think?

Reply:
You're right, better being on the safe side
I'll change it to sleep for 2 seconds each iteration and have 30 retries (to sum up to a minute)